src/native-automation/request-pipeline/index.ts

```diff
@@ -12,7 +12,7 @@ import CertificateErrorEvent = Protocol.Security.CertificateErrorEvent;
 import HeaderEntry = Protocol.Fetch.HeaderEntry;
 import NativeAutomationRequestHookEventProvider from '../request-hooks/event-provider';
 import ResourceInjector, { ResourceInjectorOptions } from '../resource-injector';
-import { convertToHeaderEntries } from '../utils/headers';
+import { convertToHeaderEntries, getHeaderValue } from '../utils/headers';
```

Comment:
please use getHeaderValue in other places (if it is applicable) such as:
```JS
private static _isPage (responseHeaders: HeaderEntry[] | undefined): boolean {
        const contentType = responseHeaders
            ?.find(header => header.name.toLowerCase() === 'content-type')
            ?.value;

        if (contentType)
            return contentTypeUtils.isPage(contentType);

        return true;
    }
```
Should we use `toLowerCase` to ensure that out comparison is correct?